### DISCUSSION TITLE
Transactions with custom scope

### DISCUSSION THREAD

**Discussion by jenscski** [getsentry/sentry-dotnet#2596] [Q&A]
Is there a way to use transactions with a custom scope?

Similar to how it's possible to use custom scope with events, https://github.com/getsentry/sentry-dotnet/discussions/2340

**Comment by jamescrosswell** [comment]
@jenscski I'm not sure I follow. I did take a look at that link, but it doesn't seem to talk about custom scopes with events.

What is it you're trying to do?

**Comment by jenscski** [comment]
I think I asked this question before I got the full understanding about transactions ... Closing
